mars/dataframe/base/tests/test_base.py

```diff
@@ -888,3 +888,24 @@ def a():
         df.eval("p, q = a + c")
     with pytest.raises(ValueError):
         df.query("p = a + c")
+
+
+def test_empty():
+    # for DataFrame
+    assert from_pandas_df(pd.DataFrame()).empty == pd.DataFrame().empty
+    assert from_pandas_df(pd.DataFrame({})).empty == pd.DataFrame({}).empty
+    assert from_pandas_df(pd.DataFrame({'a':[]})).empty == pd.DataFrame({'a':[]}).empty
+    assert from_pandas_df(pd.DataFrame({'a':[1]})).empty == pd.DataFrame({'a':[1]}).empty
+    assert from_pandas_df(pd.DataFrame({'a':[1], 'b':[2]})).empty == pd.DataFrame({'a':[1], 'b':[2]}).empty
+    assert from_pandas_df(pd.DataFrame(np.empty(shape=(4,0)))).empty == pd.DataFrame(np.empty(shape=(4,0))).empty
+
+    # for Series
+    assert from_pandas_series(pd.Series()).empty == pd.Series().empty
+    assert from_pandas_series(pd.Series({})).empty == pd.Series({}).empty
+    assert from_pandas_series(pd.Series({'a':[]})).empty == pd.Series({'a':[]}).empty
+    assert from_pandas_series(pd.Series({'a':[1]})).empty == pd.Series({'a':[1]}).empty
+
+    # Maybe fail due to lazy evaluation
+    with pytest.raises(RuntimeError):
+        a = from_pandas_df(pd.DataFrame(np.random.rand(10, 2)))
+        assert a[a>0].empty == True
```

Comment:
Add a new line at end of file

Reply:
Have done it but don't know why we need it

Reply:
It seems that I also need to modify a[a>0] to a[a > 0], because of [ubuntu-latest, 3.9-cython] tests?

Reply:
We use Flake8 for code linting, it breaks the rule `No newline at end of file (W292)`.

Reply:
Then let me fix this and `reset` to make the log a bit cleaner